### PR TITLE
[Issue-145] CSI should handle OpenShift system disks

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
## Purpose
Solves https://github.com/dell/csi-baremetal/issues/145

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved
- [x] PR validation is passed
- [x] Custom CI is passed

## Testing
_Link to custom CI build_

### CODE REVIEW

File: charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml [hunk 1]
```diff
@@ -233,6 +236,10 @@ spec:
         hostPath:
           path: /sys
           type: Directory
+      - name: host-root
```
Comment (rususlasan) [inline charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:239]: Let's make it configurable and do not share host root by default.

Comment (AnastasiaShemyakinskaya) [reply charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:242]: Done

File: pkg/base/constants.go [hunk 2]
```diff
@@ -41,6 +41,10 @@ const (
 
 	// KubeletRootPath is the pods' path on the node
 	KubeletRootPath = "/var/lib/kubelet/pods"
+
+	// OpenShiftHostRootPath is root mount in pods for OpenShift cluster
+	OpenShiftHostRootPath = "/hostroot/sysroot"
```
Comment (mishoyama) [inline pkg/base/constants.go:46]: I prefer to avoid having names of particular platforms in the code.

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: charts/baremetal-csi-plugin/values.yaml [hunk 3]
```diff
@@ -4,6 +4,7 @@ global:
 
 env:
  test: false
+ openshift: false
```
Comment (mishoyama) [inline charts/baremetal-csi-plugin/values.yaml:7]: please rename to `mountHostRoot` since this problem might occur on any platform.

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: pkg/base/constants.go (hunk 2 above)
Comment (AnastasiaShemyakinskaya) [reply pkg/base/constants.go]: Fixed

File: charts/baremetal-csi-plugin/values.yaml (hunk 3 above)
Comment (AnastasiaShemyakinskaya) [reply charts/baremetal-csi-plugin/values.yaml]: Fixed

**Review by mishoyama — APPROVED** [review]

**Review by rususlasan — APPROVED** [review]

**Review by painhardcore — APPROVED** [review]

**Pull request merged**